operator/pkg/install/install.go

```diff
@@ -0,0 +1,337 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
```

Comment:
Still have some hope we can get rid of most/all of this and use `helm install` more directly (maybe with `istioctl helm-install` if back compat makes this impossible to do transparently with the existing stuff)